gymnax/wrappers/gym.py

```diff
@@ -68,7 +68,7 @@ def step(
         o, self.env_state, r, d, info = self._env.step(
             step_key, self.env_state, action, self.env_params
         )
-        return o, r, d, d, info
+        return o, r, d, info
```

Comment:
@keertanavc Not sure about this change here, because Gymnax is using the new API of gym>=0.26.0 that introduced a breaking API change to the `step` function (see [here](https://gymnasium.farama.org/main/release_notes/gym_v0.26.0/)) where `done` can be `truncated | terminated`, so there should be two booleans returned.

Though, judging by the typing of the function, it is not clear that the `step` function here in Gymnax is assuming the new gym API.